idea_town/frontend/static-src/app/templates/experiment-page.js

```diff
@@ -6,9 +6,9 @@ export default `
         <div class="details-header">
           <h1 data-hook="title"></h1>
           <div class="idea-controls">
-            <button data-hook="feedback" id="feedback-button" class="button primary">Give Feedback</button>
-            <button data-hook="uninstall" id="uninstall-button" class="button neutral"><span class="state-change-inner"></span><span class="transition-text">Disabling...</span><span class="default-text">Disable <span data-hook="title"></span></span></button>
-            <button data-hook="install" id="install-button" class="button primary"><span class="state-change-inner"></span><span class="transition-text">Enabling...</span><span class="default-text">Enable <span data-hook="title"></span></span></button>
+            <button data-l10n-id="giveFeedback" data-hook="feedback" id="feedback-button" class="button primary">Give Feedback</button>
+            <button data-hook="uninstall" id="uninstall-button" class="button neutral"><span class="state-change-inner"></span><span data-l10n-id="disableExperimentTransition" class="transition-text">Disabling...</span><span data-l10n-id="disableExperiment" class="default-text">Disable <span data-hook="title"></span></span></button>
```

Comment:
> https://blog.mozilla.org/l10n/2012/07/16/l20n-features-explained-dom-overlays/

@zbraniecki Thanks for the link. DOM overlays look like _exactly_ what I would want from an l10n library, nice work. (FWIW, I was looking (and not finding) over at http://l20n.org/learn.)